pkg/api/version.go

```diff
@@ -1,20 +1,3 @@
-// Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
I wonder why this was removed

Reply:
I think the swagger generation may not have run fully. The SDK update seems to not compile anymore due to some missing endpoints.

I think when the code compiles it will also re-add the licenses.